modules/multi-architecture-migrating-cluster.adoc

```diff
@@ -0,0 +1,36 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/cluster-tasks.adoc
+:_content-type: PROCEDURE
+[id="multi-architecture-migrating-cluster_{context}"]
+
+= Migrating to a cluster with multi-architecture compute machines 
+
+You can use the following procedure to upgrade existing single-architecture cluster to one that supports multi-architecture compute machines.
+
+[NOTE]
+====
+The Cluster Samples Operator is not supported on clusters with multi-architecture compute machines. You can disable Cluster Samples Operators in your cluster capabilities. 
+==== 
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`). 
+
+.Procedure
+* To upgrade your existing single-architecture cluster, run the following command: 
+[source,terminal]
++
+----
+$ oc adm upgrade --to-multi-arch
+----
+
```

Comment:
IMO we tend to overuse admonitions. How about this:

1.  Move `The migrations must be complete and all the cluster operators must be stable before you can add compute machine sets with different architectures to your cluster. ` directly following your single-step procedure.
2. Add a Verification section and move the content around monitoring the upgrade process there.